client/src/app/site/cinema/components/cinema/cinema.component.html

```diff
@@ -3,11 +3,12 @@
         <h2>{{ 'Autopilot' | translate }}</h2>
     </div>
 </os-head-bar>
-
-<!-- Title Card -->
-<mat-card class="os-card">
-    <h1 class="line-and-icon">
-        {{ title }}
+<div class="content-container">
+    <!-- Title Card -->
+    <mat-card class="os-card">
+        <h1 class="line-and-icon">
+            {{ title }}
+        </h1>
```

Comment:
results in the following:
![image](https://user-images.githubusercontent.com/10233032/98130617-bedf7d00-1eba-11eb-8ae7-f625f7d4495c.png)

Reply:
You're right. Done.